test/e2e/performanceprofile/functests/utils/cgroup/v2/v2.go

```diff
@@ -77,7 +77,7 @@ func (cm *ControllersManager) CpuStat(ctx context.Context, pod *corev1.Pod, cont
 	// cpu.stat always contains 3 stats usage_usec, user_usec, system_usec
 	// only when cpu controller is enabled other stats like nr_periods etc are enabled
 	if len(interfacevalues) < 4 {
-		return nil, fmt.Errorf("CPU Controller is not enabled")
+		return nil, fmt.Errorf("cpu.stat doesn't contain all the cpu metrics required")
```

Comment:
minor: this was restored in the later commit:
https://github.com/openshift/cluster-node-tuning-operator/pull/955/commits/8092af08081d8334505aaa3bc148f2c221a24cba#diff-f2536864a27be1f85973bce12efd1a1447d9fab4bb03170997c538c5e5650e81R81
if not intended to change it here, I think better to discard this line change

Reply:
fixed in the latest commit